analyses/molecular-subtyping-pathology/02-incorporate-clinical-feedback.Rmd

```diff
@@ -0,0 +1,200 @@
+ 
+**UPDATE 2020-10-05** 
+Below is a double-reviewed (by Cassie Kline and myself) table of histone mutations found in the specified patient tumor per the TGEN genomic reports. 
+
+Kids_First_Participant_ID | H3 Status | tumor_descriptor
+-- | -- | --
+PT_M23Q0DC3 | H3F3A K28M | Initial CNS Tumor
+PT_9GKVQ9QS | H3F3A K28M | Initial CNS Tumor
+PT_KBFM551M | H3 WT | Initial CNS Tumor
+PT_KBFM551M | no report | Progressive   Disease Post-Mortem
+PT_V1HNAC2Q | H3F3A K28M | Initial CNS Tumor
+PT_NK8A49X5 | H3F3A K28M | Initial CNS Tumor
+PT_NK8A49X5 | H3F3A K28M | Progressive
+PT_KZ56XHJT | H3F3A K28M | Initial CNS Tumor
+PT_KZ56XHJT | H3F3A K28M | Progressive
+PT_KZ56XHJT | no report | Progressive   Disease Post-Mortem
+PT_QA9WJ679 | H3F3A K28M | Initial CNS Tumor
+PT_WGVEF96B | H3F3A K28M | Initial CNS Tumor
+PT_HGM20MW7 | H3F3A K28M | Initial CNS Tumor
+PT_0MXPTTM3 | H3 WT | Initial CNS Tumor
+PT_M9XXJ4GR | H3F3A K28M | Initial CNS Tumor
+PT_KTRJ8TFY | H3F3A K28M | Initial CNS Tumor
+PT_KTRJ8TFY | no report | Progressive   Disease Post-Mortem
+PT_1E3E6GMF | H3F3A K28M | Initial CNS Tumor
+PT_VPEMAQBN | HIST1H3B K28M | Initial CNS Tumor
+PT_7P6J57H3 | H3F3A K28M | Initial CNS Tumor
+PT_C5FKRB1P | H3 WT | Initial CNS Tumor
+PT_CSKHQB16 | HIST1H3B K28M | Initial CNS Tumor
+PT_1AAYYGGY | H3F3A K28M | Initial CNS Tumor
+PT_NWYSD53S | H3F3A K28M | Initial CNS Tumor
+PT_8P368R5B | No report | Initial CNS Tumor
+PT_C9YDTZPA | H3F3A K28M | Initial CNS Tumor
+PT_KAQMYFYB | H3F3A K28M | Initial CNS Tumor
+PT_Y74CVASJ | H3F3A K28M | Initial CNS Tumor
+PT_W5GP3F6B | H3F3A K28M | Initial CNS Tumor
+PT_A06JR0E5 | H3F3A K28M | Initial CNS Tumor
+PT_EN2RN5Y1 | H3F3A K28M | Initial CNS Tumor
+PT_DMAF1J4A | H3F3A K28M | Initial CNS Tumor
+PT_RYMG3M91 | H3F3A K28M | Initial CNS Tumor
+PT_RE6AXQM1 | H3F3A K28M | Initial CNS Tumor
+PT_G16VK7FR | HIST1H3B K28M | Initial CNS Tumor
+PT_1YQH5NSH | H3F3A K28M | Initial CNS Tumor
+
+## Set up 
+
+```{r}
+# This logic is the same as analyses/gene-set-enrichment-analysis/02-model-gsea.Rmd
+# Assigning params$is_ci to running_in_ci avoids a locked binding error
+running_in_ci <- params$is_ci
+
+# Are we testing? In case of a non 0/1 number, we recast as logical, and then 
+# ensure logical.
+if (running_in_ci %in% c(0,1)) running_in_ci <- as.logical(running_in_ci)
+if (!(is.logical(running_in_ci)))
+{
+  stop("\n\nERROR: The parameter `is_ci` should be FALSE/TRUE (or 0/1).")
+}
+```
+
+
+### Directory and files
+
+#### Directories
+
+```{r}
+root_dir <- rprojroot::find_root(rprojroot::has_dir(".git"))
+data_dir <- file.path(root_dir, "data")
+results_dir <- "results"
+```
+
+
+#### Input
+
+When we run this locally, we want to tie it to a specific version of the histologies file _prior_ to reviewed clinical subtypes for PNOC003 samples and pathology feedback [#751](https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/751)
```

Comment:
Hmmmm... _do_ we want to tie this to a specific release? 🤔  I believe the rationale for tying `analyses/molecular-subtyping-pathology/01-compile-subtyping-results.Rmd` to a specific release was to preserve the `integrated_diagnosis`, `broad_histology`, and `short_histology` fields before they got updated. But now, all of the subtyping uses `pathology_diagnosis` and `pathology_free_text_diagnosis` fields and `integrated_diagnosis`, `broad_histology`, and `short_histology` will be downstream of subtyping (#748). Given that we're going to redo the `01` notebook (#784), I expect tying that to v15 very well will come out.

What was your rationale for tying this to v15? Was it something besides consistency with the rest of the module?

Reply:
just wanted to keep it consistent in the module, forgot to mention this in the PR

Reply:
Gotcha, I think it's okay to take it out at this point. That will also allow you to simplify things (always good!) since there won't be a need to distinguish between running in CI and running locally.